.github/workflows/labeler.yaml

```diff
@@ -4,11 +4,12 @@ on:
 
 jobs:
   triage:
+    name: Label Triage
     permissions:
       contents: read
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v4
+    - uses: actions/labeler@v5.0.0
```

Comment:
Isn't that better to point to major version only? The `actions/labeler` follows the GitHub recommendations about actions versioning, so the v5 tag is available as well.

Reply:
Yes I have changed it to the major version v5. Waiting for the checks to pass. Thanks!